app/config/schema/config.proto

```diff
@@ -0,0 +1,216 @@
+    bool use_local_sessions = 6;
+
+    // Options related to space moderators
+    // Optional.
+    Moderation moderation = 7;
+
+    // ID of the room to place users when they first join
+    // Required.
+    string start_room = 8;
+
+    // Default display name in Jitsi video calls
+    // Required.
+    string video_display_name = 15;
+
+    // Options related to username & avatar selection on the welcome page
+    // Required.
+    WelcomePage welcome_page = 9;
+
+    // Sets of avatar choices
+    // Each AvatarSet should have the same number of images
+    // Required if "avatarSelectionEnabled" is true in "welcomePage".
+    repeated AvatarSet avatars = 10;
+
+    // Set of additional avatars for specific situations
+    // Correspond to the user's choice of avatar
+    // Optional
+    SpecialAvatarSets special_avatars = 11;
+
+    // Options related to displaying the map
+    // Required.
+    MapConfig map = 12;
+
+    // Options related to the poking feature
+    // Optional. If not provided, poking is turned off.
+    PokeConfig poke = 13;
+
+    // Number of server-side proxies in use (if unsure, probably 0)
+    // Required.
+    int32 num_proxies = 14;
+}
+
+message Moderation {
+    // Contact emails for moderators to receive moderation form submissions
+    // TODO: Is this currently used?
+    // Required (at least one).
+    repeated string moderator_emails = 1;
+
+    // Moderator cell phone number to receive text of form submissions
+    // Required.
+    string moderator_number = 2;
+
+    // Description shown to users when sending moderator messages
+    // Required.
+    string form_description = 3;
+}
+
+message WelcomePage {
+    // Whether to have users select an avatar, or use a default for everyone
+    // Optional. Defaults to false.
+    bool avatar_selection_enabled = 1;
+
+    // Image to show behind the welcome page
+    // Optional.
+    string background_image_path = 2;
+
+    // What text to show on the button to join the party
+    // Required.
+    string enter_space_button_text = 3;
+
+    // Required.
+    string avatar_selection_text = 4;
+
+    // Optional.
+    string header_html = 5;
+
+    // Configuration for a sign-in iframe before joining the space.
+    // Optional.
+    Auth auth = 6;
+}
+
+message Auth {
+    // Path to the login page to iframe
+    // Optional.
+    string iframe = 1;
+
+    // HTML to display with the login page
+    string description_html = 2;
+}
+
+message AvatarSet {
+    // Paths to avatar images
+    // Required.
+    repeated string images = 1;
+}
+
+message SpecialAvatarSets {
+    // The set of avatars to show when a user exits
+    // Should be the same size as the regular avatar sets
+    // Optional.
+    AvatarSet goodbye = 1;
+}
+
+message MapConfig {
+    // Options related to the stats to display above the map
+    // Required.
+    MapStats stats = 1;
+
+    // TODO: What is this, and what values can it take?
+    string scale_by = 2;
```

Comment:
@morganecf @hndrewaall help <3

Reply:
This was also a last-minute pre-party hack…it's supposed to control whether or not the map room colors are scaled by opacity or by a d3 color scale, but I think I ended up just panic-commenting the color scale logic out of `MapVisualization.jsx`, so it's not even used. Let's get rid of it from the config!

Reply:
Cool, removed this